metabase/metabase
# fix sending test subscriptions on dashboards with actions

Closes https://github.com/metabase/metabase/issues/28426

### Description

Filter out actions cards when sending test dashboard subscriptions

### How to verify

1. Run an instance where you create actions
2. Create a dashboard with a regular card and an action
3. Try sending a test subscription, ensure it works

### Checklist
